hci_cyw43439.go

```diff
@@ -0,0 +1,27 @@
+//go:build cyw43439
```

Comment:
Does this only work for CYW43439?

Reply:
Yes, the hci packet opcodes are board specific